docs/reference_utils.rst

```diff
@@ -7,3 +7,14 @@ The wgpu library provides a few utilities. Note that the functions below need to
 .. autofunction:: wgpu.utils.get_default_device
 
 .. autofunction:: wgpu.utils.compute_with_buffers
+
+
+Shadertoy
+--------------------
+
+The Shadertoy class provides a "screen pixel shader programming interface" similar to `shadertoy <https://www.shadertoy.com/>`_,
+Helps you research and quickly build or test shaders using WGSL via WGPU.
```

Comment:
Bit of a nitpick, but could you put this explanation in the class docstring instead?